### PR TITLE
salt 3003

### PR DESCRIPTION
**Pull request by SMillerDev**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/salt.rb [hunk 1]
```diff
@@ -172,6 +167,571 @@ class Salt < Formula
...
+  resource "pyobjc-framework-Vision" do
+    url "https://files.pythonhosted.org/packages/67/2f/4f68e5b553761197d288e98821eeb6f1e6a220cb0fe78c7fd68d8de575f6/pyobjc-framework-Vision-6.2.2.tar.gz"
+    sha256 "dc03fea57cc52ba15a2b39be6e834cfd49a6e28daf9b862a578a8002a6fc5207"
+  end
+
+  resource "pyobjc-framework-WebKit" do
+    url "https://files.pythonhosted.org/packages/64/19/f26eb6b607aec27b3adcd6387e510466e113da1502ccea8069691ad4c642/pyobjc-framework-WebKit-6.2.2.tar.gz"
+    sha256 "87e9409ace6d2e29c776bd76df75fe922a1f2a225eef815640c6047d25da8dab"
+  end
+
```
Comment (carlocab) [inline Formula/salt.rb:734]: 😅

**Comment by carlocab** [comment]
```
pkg_resources.DistributionNotFound: The 'pyobjc==6.2.2' distribution was not found and is required by salt
```

**Comment by Bo98** [comment]
You'll need to strip out the `pyobjc` line from `requirements/darwin.txt`.

**Comment by SMillerDev** [comment]
Ready for review (again)

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**